components/CreateOrganizationForm.js

```diff
@@ -53,19 +53,19 @@ const orgMessages = defineMessages({
   },
   errorSlugHyphen: {
     id: 'createOrg.form.error.slug.hyphen',
-    defaultMessage: 'Organization slug can not start nor end with hyphen',
+    defaultMessage: 'Organization URL slug cannot start or end with a hyphen (-)',
   },
   errorWebsite: {
     id: 'createOrg.form.error.website',
-    defaultMessage: 'Enter valid website format www.test.com or test.org',
+    defaultMessage: 'Enter valid a website, e.g. www.example.com or example.org',
```

Comment:
I think should be; 

```suggestion
    defaultMessage: 'Enter a valid website, e.g. www.example.com or example.org',
```

Reply:
OK that's fine. It's just that we have so many kinds of connected accounts in different areas of the settings, but this section appears to be only for Twitter connection. But it's not very important.